normalized-cache-incubating/src/commonMain/kotlin/com/apollographql/cache/normalized/api/Record.kt

```diff
@@ -22,6 +22,7 @@ class Record(
      * - CacheKey (for composite types)
      * - Map (for custom scalars)
      * - null
+     * - Error (for GraphQL errors)
```

Comment:
This could be removed here as it's already part of the `RecordValue` typealias (avoid maintaining this in 2 places)